spec.html

```diff
@@ -24798,12 +24836,18 @@ <h2>Syntax</h2>
         ExportSpecifier :
           IdentifierName
           IdentifierName `as` IdentifierName
+          IdentifierName `as` ModuleExportName
+          ModuleExportName
+          ModuleExportName `as` ModuleExportName
```

Comment:
It should also be possible to rename `ModuleExportName` to `IdentifierName` when re‑exporting:
```suggestion
          ModuleExportName `as` IdentifierName
          ModuleExportName `as` ModuleExportName
```